save-core/src/commonMain/kotlin/org/cqfn/save/core/Save.kt

```diff
@@ -22,8 +23,11 @@ class Save(
         requireNotNull(testConfig) { "Provided path ${saveProperties.testConfig} doesn't correspond to a valid save.toml file" }
 
         val plugins: List<Plugin> = emptyList()  // todo: discover plugins (from configuration blocks in TestSuiteConfig?)
+        logInfo("Discovered plugins: $plugins")
         plugins.forEach {
+            logInfo("Execute plugin: ${it::class::simpleName}")
             it.execute(saveProperties, testConfig)
+            logInfo("${it::class::simpleName} successfully executed!")
```

Comment:
```suggestion
            logInfo("${it::class.simpleName} successfully executed!")
```
I believe class name is a property, not a reference